static/app/views/alerts/issueRuleEditor/ruleNode.tsx

```diff
@@ -71,7 +71,9 @@ class RuleNode extends React.Component<Props> {
           initialVal = fieldConfig.choices[0][0];
         }
       } else {
-        initialVal = data[name];
+        initialVal = isNaN(parseInt(data[name] as string, 10))
+          ? data[name]
+          : parseInt(data[name] as string, 10);
```

Comment:
@getsentry/core-ui It looks like the new SelectControl options (rather than choices) are pretty strict about numbers in a different way than was already done. Was this intentional?

We store all these fields as strings, but when we select, we store them as numbers so that mismatch didn't allow default values to appear

Reply:
cc @davidenwang